doc/api_base.rst

```diff
@@ -97,6 +97,10 @@
 
      Configure plugins by running their configure() method.
 
+  .. method:: unload_plugins()
+
+     Unload plugins by running their unload() method.
```

Comment:
I think this is not correct and could be misleading.
Plugins don't have an `unload()` method that is called for each plugin. This is different from eg. [`configure()`](https://github.com/rpm-software-management/dnf-plugins-core/blob/master/plugins/builddep.py#L106) or [`pre_configure()`](https://github.com/rpm-software-management/dnf-plugins-core/blob/master/plugins/builddep.py#L102) those are actually called.